TrafficCapture/captureKafkaOffloader/src/main/java/org/opensearch/migrations/trafficcapture/kafkaoffloader/KafkaCaptureFactory.java

```diff
@@ -8,16 +8,22 @@
 import org.opensearch.migrations.trafficcapture.IChannelConnectionCaptureSerializer;
 import org.opensearch.migrations.trafficcapture.IConnectionCaptureFactory;
 import org.opensearch.migrations.trafficcapture.StreamChannelConnectionCaptureSerializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
```

Comment:
See the comment at the top-level.  Since MDC is handled as a ThreadLocal, this is going to be tech-debt that is extremely difficult to track and diagnose.  Case in point, there aren't any tests in place to verify that the right values are going to be coming back out into the logs.

From a logging config management perspective, the MDC key/values can be handled fluently.  See https://chat.openai.com/share/08649025-20e3-42e6-abba-32c29722dc0c and https://github.com/opensearch-project/opensearch-migrations/pull/280